feature-policy/resources/featurepolicy.js

```diff
@@ -1,3 +1,7 @@
+// Feature test to avoid timeouts
+function assert_feature_policy_supported() {
+  assert_not_equals(document.policy, undefined, 'document.policy');
```

Comment:
Maybe "Feature Policy is supported" as the description, since this isn't really about testing document.policy, and making it trivial to understand the failure reason?

Reply:
fixed